README.md

```diff
@@ -136,6 +136,9 @@ Following the portal logs is one way to monitor its startup progress:
 docker logs -f portal-service
 ```
 
+### Update tips
+You should of course `git pull` compose-services if you have not done so for a while. You also need to `docker pull` new images from Quay--this will not happen automatically. If your git pull pulled new commits, and you already have a `Secrets` folder, you may also need to delete your old `Secrets` and rerun `creds_setup.sh` (see [Setting up Credentials](#Setting-up-Credentials)) to recreate it. 
```

Comment:
I would recommend `docker-compose pull` - which will pull the latest bytes for all the images in `docker-compose.yml`